gradle.properties

```diff
@@ -5,7 +5,7 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -XX:+CMSPe
 bundleInstallRoot = /apps/grabbit/install
 
 group = com.twcable.grabbit
-version = 6.0.0
```

Comment:
Oh yeah for sure. I normally just have it for testing. Will be removing it from this commit before TM.